app/terminal/terminal.py

```diff
@@ -128,6 +121,16 @@ def _flush(self, content):
         content = colored(content, 'yellow')
         self._sout.write(str(content) + '\n')
 
+    def _add_facts_from_csv(self, csv_path, op_id):
```

Comment:
since we're back to a single function, let's put this _add_facts_from_csv logic back under the do_up function